mne/io/reference.py

```diff
@@ -321,13 +322,19 @@ def set_eeg_reference(inst, ref_channels='average', copy=True,
             # Creating an average reference may fail. In this case, make
             # sure that the custom_ref_applied flag is left untouched.
             custom_ref_applied = inst.info['custom_ref_applied']
+
+            if len(ch_type) > 1:
```

Comment:
Dear @agramfort, thank you very much for working on this! In my case, I would actually love to use the average reference projection for multiple channel types. 

My patient dataset has both ECOG strips on the cortex and DBS strips in deep brain structures. I would like to apply an average reference projection for both channel types `ch_type=('ecog', 'dbs')` together. It seems, you would like to explicitly **exclude** this possibility. 

Would it be possible to **include** this possibility?